portfolio/src/main/java/com/google/sps/servlets/NewServlet.java

```diff
@@ -8,12 +8,12 @@
 import javax.servlet.http.HttpServletResponse;
 
 
-@WebServlet("/servlet")
+@WebServlet("/step_one_servlet")
```

Comment:
Try a more descriptive name -- step_one_servlet does not let me know the function of this endpoint.